permission.php

```diff
@@ -95,4 +95,9 @@
     // Core
     "logfiles"          => "Beheer logbestanden",
```

Comment:
This file is missing 2.4.0 to 3.0.0 translations

Reply:
Added missing strings and completed translation of this file